tests/examples/wibox/layout/grid/add_row_border1.lua

```diff
@@ -0,0 +1,85 @@
+--DOC_GEN_IMAGE --DOC_HIDE_START
+local generic_widget_ = ...
+local wibox     = require("wibox")
+local beautiful = require("beautiful")
+
+local l = wibox.widget {
+    spacing = 10,
+    layout  = wibox.layout.fixed.horizontal
+}
+
+local function generic_widget(txt)
+    return generic_widget_(txt, nil, 0)
+end
+
+--DOC_HIDE_END
+   local w = wibox.widget {
+       --DOC_HIDE_START
+       -- [...] Some widgets here.
```

Comment:
I think you want to reverse these 2 lines.

Reply:
done